bot_cli/commands.py

```diff
@@ -264,3 +263,43 @@ def birthdays(args):
         print_success(result)
     else:
         print_warn(f"There is no one to celebrate birthday for next {period} day(s)")
+
+
+@contact_not_found_error
+@add_address_error
+def add_address(args):
+    try:
+        name, *address = args
+        name = Name(name)
+        address = " ".join(address)
+    except:
+        raise CommandError
+
+    if address_book.find(name):
```

Comment:
Separate variable for contact to not search twice in case it was found.

Reply:
Done